packages/sdk/src/actor/actorInternal.ts

```diff
@@ -40,11 +40,11 @@ export class ActorInternal implements InternalPatchable<ActorLike> {
 		const behavior = (this.behavior && this.behavior.behaviorType === actionEvent.behaviorType)
 			? this.behavior : undefined;
 		if (behavior && behavior._supportsAction(actionEvent.actionName)) {
-			behavior._performAction(actionEvent.actionName, actionEvent.actionState, actionEvent.user);
+			behavior._performAction(actionEvent.actionName, actionEvent.actionState, actionEvent.user, actionEvent.actionData);
```

Comment:
This line causes a lint error (max line length). Please run a lint check. There are a few new warnings in this changeset.

Reply:
Didn't realize that the lint was separate from build now.